src/context.cpp

```diff
@@ -579,6 +580,33 @@ namespace librealsense
             software_sensor::close();
         }
 
+        void add_option( const realdds::dds_option & option )
+        {
+            //Convert name to rs2_option type
+            rs2_option opt_type = RS2_OPTION_BACKLIGHT_COMPENSATION;
```

Comment:
What is this? initialization to the first option?

Reply:
Yes.
I can also cast 0 to rs2_option, but I like this style less

Reply:
I don't like it as it looks like a special handling, and if the for loop cannot find the option it handles the init value? sounds wrong.
I suggest something else.
Setting to -1 (or `RS2_OPTION_COUNT`)
and after the for loop add a condition that throw or something if it stayed the same.
Thoughts?

Reply:
I will use 0 and cast. [Like in this example](https://github.com/IntelRealSense/librealsense/blob/master/examples/sensor-control/api_how_to.h#L154)